### ISSUE TITLE
Single-column ResultSet gets truncated on empty string

### ISSUE THREAD

**Issue by Aivean** [yandex/clickhouse-jdbc#34]
Here is the minimal example:

```
 select c from (
    select 'a' as c
    UNION ALL select 'b' as c
    UNION ALL select '' as c
    UNION ALL select 'd' as c
 )
```

ResultSet is read using something like this:

```
val s = conn.createStatement()
val r = s.executeQuery(sql)

while (r.next()) {
   println(r.getString(1))
}
```

Expected result should be:
```
a
b

d
```

Actual result is:
```
a
b
```

Here is the full test snippet (scala): 
https://gist.github.com/Aivean/fe8c4f90c183f08aa908baed7ca89478

P.S. Both TCP and HTTP interfaces return correct result.

**Comment by serebrserg** [comment]
Thanks for report. I will check.

**Comment by serebrserg** [comment]
Fixed in 0.1.12. Will be available in maven central soon.

**Comment by Aivean** [comment]
Awesome, thanks.
